packages/gatsby/src/schema/infer-graphql-input-fields.js

```diff
@@ -264,7 +264,7 @@ export function inferInputObjectStructureFromNodes({
     let key = k
     // Remove fields for traversing through nodes as we want to control
     // setting traversing up not try to automatically infer them.
-    if (isRoot && EXCLUDE_KEYS[key]) return
+    if (typeof value === `symbol` || (isRoot && EXCLUDE_KEYS[key])) return
```

Comment:
Could we import `INVALID_VALUE` from `data-tree-utils.js` and compare directly against that? We shouldn't have any other symbols at this point (and same treatment for change in other file)

Reply:
@pieh sounds good. Unfortunately I don't have the bad data anymore to verify it but when I had the bad data this fix worked to catch any generic Symbol.  I didn't inspect the actual Symbol's value so I have no idea if it was in fact an `INVALID_VALUE`